.gitlab-ci.yml

```diff
@@ -286,6 +286,7 @@ jobstore_and_provisioning:
   script:
     - pwd
     - ${MAIN_PYTHON_PKG} -m virtualenv venv && . venv/bin/activate && pip install -U pip wheel && make prepare && make develop extras=[all] packages=htcondor
+    - make test tests=src/toil/test/lib/aws/
```

Comment:
`TOIL_SKIP_DOCKER=true make test tests=src/toil/test/lib` is run `basic_tests`.  We could probably separate these more cleanly.